at_commons/lib/src/utils/at_key_regex_utils.dart

```diff
@@ -22,11 +23,16 @@ class Regexes {
       '''((cached:)(@(?<sharedWith>($_charsInAtSign|$_allowedEmoji){1,55}):))(_*(?<entity>($_charsInEntity|$_allowedEmoji)+))\\.(?<namespace>$_charsInNamespace)@(?<owner>($_charsInAtSign|$_allowedEmoji){1,55})''';
   static const cachedPublicKey =
       '''(?<visibility>(cached:public:){1})((@(?<sharedWith>($_charsInAtSign|$_allowedEmoji){1,55}):))?(?<entity>($_charsInEntity|$_allowedEmoji)+)\\.(?<namespace>$_charsInNamespace)@(?<owner>($_charsInAtSign|$_allowedEmoji){1,55})''';
+  static const reservedKey =
+      '^($commitLogCompactionKey|$accessLogCompactionKey|$notificationCompactionKey|$AT_PKAM_PRIVATE_KEY|$AT_PKAM_PUBLIC_KEY|$AT_ENCRYPTION_SELF_KEY|$AT_CRAM_SECRET|$AT_CRAM_SECRET_DELETED|$AT_SIGNING_KEYPAIR_GENERATED)\$';
 }
 
 class RegexUtil {
   /// Returns a first matching key type after matching the key against regexes for each of the key type
   static KeyType keyType(String key) {
+    if (reservedKeys.contains(key)) {
+      return KeyType.reservedKey;
```

Comment:
Is what specific reserved key be included?

Reply:
I don't see a need for that right now?